c/include/libsbp/system.h

```diff
@@ -82,12 +82,38 @@ typedef struct SBP_ATTR_PACKED {
  * The INS status message describes the state of the operation
  * and initialization of the inertial navigation system. 
  */
-#define SBP_MSG_INS_STATUS   0xFF03
+#define SBP_MSG_INS_STATUS            0xFF03
 typedef struct SBP_ATTR_PACKED {
   u32 flags;    /**< Status flags */
 } msg_ins_status_t;
 
 
+/** Experimental telemetry message
+ *
+ * The CSAC telemetry message has an implementation defined telemetry string
+ * from a device. It is not produced or available on general Swift Products.
+ * It is intended to be a low rate message for status purposes.
+ */
+#define SBP_MSG_CSAC_TELEMETRY        0xFF04
+typedef struct SBP_ATTR_PACKED {
+  u8 id;           /**< Index representing the type of telemetry in use.  It is implemention defined. */
+  char telemetry[0]; /**< Comma separated list of values as defined by the index */
```

Comment:
@dzollo Are all the telemetry values 8 bit values?

Reply:
Oh, I see, comma separated.

Reply:
This feels unnecessarily loosely typed to me - don't we already know all the telemetry values that are produced? Can we not make this much more strongly typed?

Reply:
we could, but this is the native data from csac and it seemed easier to just leave it as is.